frontend/next.config.js

```diff
@@ -49,7 +50,7 @@ const securityHeaders = [
   },
 ];
 
-module.exports = {
+module.exports = nextTranslate({
```

Comment:
Thank you so much! I think it's really great! 

I'm a little bit bit concerned with this package. It doesn't have that many downloads. Do you think it is a potential security vulnerability that you wrap this whole thing in nextTranslate? 

I'm looking at some other projects. It seems like they are using next-i18next. For example, cal.com (https://github.com/calcom/cal.com/blob/main/apps/web/next.config.js). It seems like it has 3 times more downloads. 

Are there any reasons why you chose next-translate over the other one?

Reply:
Other than that it looks awesome! 🔥🚀

Reply:
Sorry, actually I haven't thought about security problem 🥲.
I used next-translate because  I used this in other project and next-translate was more convenient to use and.
For example, we don't need to write getStaticSideProps or getServerSideProps in every pages.
Also, I think nextTranslate source code, https://github.com/aralroca/next-translate/blob/master/src/plugin/index.ts looks just normal next.js plugin.
But I don't have any security background and If you worry, I can change next-translate into next-i18next.

Reply:
Yeah, I'm really sorry for asking you to change it... 😭 If you think you could switch it to next-i18next, this would be awesome! Do you think it would take you a lot of time? 

I think we just need to be very cautious because of how sensitive the data Infisical stores is. If there is a security vulnerability, this could lead to many people losing their secrets and data. So i'm a bit worried that no big projects are using next-translate, right? With next-i18next, I found that Cal.com is using them which is why it seems more credible. Super sorry again 😭

Reply:
Hi @gangjun06! Just wanted to check in. Let me know if you need any help at all! :)

Reply:
@mv-turtle  Oops! Sorry for big late.
I didn't push after change library and I didn't care about it for a week.
Also there're some error apply library.
I'll fix them and make mergeable ASAP!

Reply:
Thank you so much @gangjun06! ❤️